js/plugin/RouteLoaderConverter.js

```diff
@@ -328,7 +329,41 @@ BR.routeLoader = function (map, layersControl, routing, pois) {
                 this.getSimplifiedCoords(this._options.simplifyTolerance)
             );
 
+            // use now the middle point between the "simplified" waypoint and the "previous" trkpt in the origin track
+
+            var trackPt_b = turf.cleanCoords(this._trackPoints.geometry);
```

Comment:
`cleanCoords` is redundant here, it's already done in [getLineStringsFromGeoJSON](https://github.com/nrenner/brouter-web/blob/0ee37f557a9a21a16343a7cf9b5d4d183356b898/js/plugin/RouteLoaderConverter.js#L305)